app/views/events/_form.html.erb

```diff
@@ -183,6 +183,9 @@
           </div>
 
           <script type="text/javascript">
+            $('#startdate').on('change', function() {
```

Comment:
You want to do this after the Document.ready(), otherwise startdate may or may not be known to the DOM by the time this code is executed